tensorflow/contrib/cmake/CMakeLists.txt

```diff
@@ -322,7 +324,11 @@ if(tensorflow_ENABLE_GRPC_SUPPORT)
   include(grpc)
   include_directories(${GRPC_INCLUDE_DIRS})
   # Place boringssl after grpc as grpc depends on boringssl.
-  list(APPEND tensorflow_EXTERNAL_LIBRARIES ${grpc_STATIC_LIBRARIES})
+  if (systemlib_GRPC)
+    list(APPEND tensorflow_EXTERNAL_LIBRARIES ${grpc_STATIC_LIBRARIES})
+  else (systemlib_GRPC)
+    list(APPEND tensorflow_EXTERNAL_LIBRARIES ${grpc_LIBRARIES})
```

Comment:
I think this line is causing the Windows build failure. AFAICT `${gprc_LIBRARIES}` isn't set when `WIN32` is true.